xml/admin_backup.xml

```diff
@@ -0,0 +1,398 @@
+tar czf <replaceable>backup-admin-node-$(date +'%Y-%m-%d').tgz</replaceable> -C / \
+    var/lib/misc/salt-master-credentials \
+    usr/share/salt \
+    etc/hosts \
+    etc/pki \
+    usr/share/caasp-container-manifests \
+    etc/salt/pki \
+    etc/caasp \
+    var/lib/misc/salt/cache \
+    etc/caasp/haproxy \
+    var/lib/misc/infra-secrets \
+    var/lib/misc/ldap \
+    var/lib/misc/ldap-config \
+    var/lib/misc/ssh-public-key \
+    etc/salt/pillar \
+    var/lib/misc/velum-secrets</command>
+ </screen>
+    </step>
+    <step>
+     <para>
+      Then, export the MariaDB database and archive it:
+     </para>
+<screen>&prompt.root;<command>docker exec $(docker ps -qf name="velum-mariadb") mysqldump \
+--password=$(cat /var/lib/misc/infra-secrets/mariadb-root-password) \
+velum_production | gzip > <replaceable>/var/tmp/db-backup-$(date +'%Y-%m-%d').sql.gz</replaceable></command>
+ </screen>
+    </step>
+    <step>
+     <para>
+      Finally, copy the created files <filename>backup-admin-node-[date].tgz</filename>
+      and <filename>/var/tmp/db-backup-[date].sql.gz</filename> to
+      a location of your choosing; in accordance with your best practices for
+      backup files.
+     </para>
+     <important>
+      <title>Sensitive Data</title>
+      <para>
+       Please note that the data backup files contain senstive information that
+       allow full cluster access and must be handled securely.
+      </para>
+     </important>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.admin.backup.admin.restore">
+   <title>Restore Procedure</title>
+   <procedure>
+    <step>
+     <para>
+      Set up a new machine for the &admin_node; by following
+      <xref linkend="sec.deploy.nodes.admin_install" />.
+     </para>
+     <para>
+      Make sure the &admin_node; is registered against &scc; and has the latest
+      packages installed from the update channel.
+     </para>
+     <para>
+      You must be able to access &dashboard; indicating all containers are
+      correctly started. There is, however, no need to complete the
+      configuration steps in &dashboard;.
+     </para>
+    </step>
+    <step>
+     <para>
+      Copy the <filename>backup-admin-node-[date].tgz</filename> and
+      <filename>db-backup-[date].sql.gz</filename> of the most recent backups
+      onto the newly created &admin_node; machine. Make sure to replace
+      <literal>[date]</literal> with the actual date subsituted during the
+      backup procedure.
+     </para>
+    </step>
+    <step>
+     <para>
+      SSH into the new &admin_node; machine and switch to the location where
+      you have stored the backup files.
+     </para>
+    </step>
+    <step>
+     <para>
+      Run the following command to unpack the data backup:
+     </para>
+<screen>&prompt.root;<command>tar xzf <replaceable>backup-admin-node-[date].tgz</replaceable> --exclude='mariadb*' -C / 2> /var/tmp/log.txt</command></screen>
```

Comment:
@Klaven Why do you exclude mariadb*? I'm asking because it still produces error code 2 and log is full of "ro filesystem" errors anyway...

Reply:
The reason being that mariadb has already generated users and passwords and those passwords are stored in the files i'm excluding. We just want to keep the new generated password files as everything uses those files to talk to mariadb.